jwt.js

```diff
@@ -189,6 +189,8 @@ function fastifyJwt (fastify, options, next) {
         if (!/^Bearer$/i.test(scheme)) {
           return next(new BadRequest('Format is Authorization: Bearer [token]'))
         }
+      } else {
+        return next(new BadRequest('Format is Authorization: Bearer [token]'))
```

Comment:
Hi,
requestVerify being the only function that use it I have moved this to the top of the requestVerify() function like this:
```js
const badRequestError = new BadRequest('Format is Authorization: Bearer [token]')
```
if you have a more explicit constant name in mind I will be glad to change it. ^^

Concerning the `{ options: { algorithm?: string } }` prefix used since `0.5.0` version do you want me to commit the little code I wrote to avoid any breaking change and an additional documentation deprecation warning or do you think that it is not needed and we should land the feature as it is now ?

Reply:
If you could add a quick if to throw if `options` is specified, it would be perfect.